example/lib/main.dart

```diff
@@ -3,40 +3,48 @@ import 'package:flutter/material.dart';
 import 'package:flutter_localizations/flutter_localizations.dart';
 import 'package:easy_localization/easy_localization.dart';
 
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
 
-void main() => runApp(EasyLocalization(child: MyApp()));
+  // load
+  await EasyLocalization.loadSavedLocale();
```

Comment:
This API looks very weird. Let's think about some way to make it cleaner.

Reply:
yeah  agree with u

Reply:
This is needed so that the app starts from first frame with the savedLocale loaded from SharedPreferences.
If we omit that and the fallbackLocale is different from the savedlocale, then the app would initially display words in fallbackLocale and then switch to savedLocale when the Sharedprefs resolve.

I have changed the api to be more natural imho. Please take a look and thank you for your feedback.
@aissat @danilofuchs

Reply:
Plz take a look #68 i removed  `loadSavedLocale()`

> I think it's because no fallbackLocale was provided. Maybe we should assert that supportedLocales and fallbackLocale is given. Or at least supportedLocales and use the supportedLocales.first as fallbackLocale if no explicit fallbackLocale is provided.
> 
> @aissat

I fixed this in #68